management-gateway/src/main/java/org/frankframework/management/bus/BusException.java

```diff
@@ -28,12 +30,12 @@
 public class BusException extends RuntimeException {
 	private static final Logger LOG = LogManager.getLogger(BusException.class);
 
-	private static final long serialVersionUID = 1L;
+	private static final @Serial long serialVersionUID = 1L;
 
 	private final @Getter int statusCode;
 
 	/**
-	 * Seen as WARNING
+	 * Seen as WARNING / HTTP Bad Request
 	 */
 	public BusException(String message) {
 		this(message, 400);
```

Comment:
Ah, dit komt op dezelfde http status neer als mijn EmptyMessage(400) suggestie vermoed ik. Laat die opmerking dan maar zitten ;-)

Reply:
Klopt, alleen is de message niet empty.